server/src/database/schema.ts

```diff
@@ -0,0 +1,33 @@
+export const CourseCode = `CREATE TABLE CourseCode (
+    CourseDept VARCHAR(200),
+    CourseNumber VARCHAR(200),
```

Comment:
For the schema overall, I think fields like CourseDept, CourseNumber, PreReqCourseDept, PreReqCourseNumber, CoReqCourseDept, CoReqCourseNumber, and maybe a few others can be reduced in size? 
Assuming the department names are just the 4-5 letter codes, and the numbers are also 3-5 digits, I think we'll be safe to set all these fields to a size much smaller than 200.

Reply:
I believe varchar doesn't take up extra space than what they actually need (correct me if I'm wrong!), however 255 seems to be what people are suggesting for performance reason. Should we go with that?

Reply:
Oh you're right, my bad!